pkg/controllers/management/kontainerdrivermetadata/data.go

```diff
@@ -46,6 +41,23 @@ const (
 
 var existLabel = map[string]string{sendRKELabel: "false"}
 
+//settings corresponding to keys in setting2.MetadataSettings
+var userUpdateSettingMap = map[string]settings.Setting{
+	"k8s-version":             settings.KubernetesVersion,
```

Comment:
Maybe would be good to declare these keys as const under say "rancher/pkg/settings" and reuse them across the store and kontainerdrivermetadata controller code?

Reply:
@kinarashah can we do similar change for store as well?